src/mvBasicWidgets.cpp

```diff
@@ -341,6 +341,7 @@ DearPyGui::fill_configuration_dict(const mvListboxConfig& inConfig, PyObject* ou
         return;
     PyDict_SetItemString(outDict, "items", mvPyObject(ToPyList(inConfig.names)));
     PyDict_SetItemString(outDict, "num_items", mvPyObject(ToPyInt(inConfig.itemsHeight)));
+	PyDict_SetItemString(outDict, "index", mvPyObject(ToPyBool(inConfig.sendindex)));
```

Comment:
In the parser, the parameter is named "index_items". Here, its name is "index". Oops.